npm/webpack-preprocessor/__snapshots__/compilation.spec.js

```diff
@@ -15,14 +14,5 @@ SyntaxError: <path>/_test-output/syntax_error_spec.js: Unexpected token (1:18)
 `
 
 exports['webpack preprocessor - e2e has less verbose "Module not found" error 1'] = `
-Webpack Compilation Error
-.<path>/_test-output/imports_nonexistent_file_spec.js
-Module not found: Error: Can't resolve './does/not-exist' in '<path>/_test-output'
-Looked for and couldn't find the file at the following paths:
-[<path>/_test-output/does/not-exist]
-[<path>/_test-output/does/not-exist.wasm]
-[<path>/_test-output/does/not-exist.mjs]
-[<path>/_test-output/does/not-exist.js]
-[<path>/_test-output/does/not-exist.json]
- @ .<path>/_test-output/imports_nonexistent_file_spec.js 1:0-26
+Webpack CompLooked for and couldn't find the file at the following paths:resolve './does/not-exist' in '<path>/_test-output'
```

Comment:
This snapshot update doesn't look right. Might be a case where we need special handling based on whether it's webpack 4 or 5.

Reply:
Updated: https://github.com/cypress-io/cypress/pull/15611/commits/9e6d46addea6c8bba2d7fa739579521ed25e6521

I added some webpack 5 specific logic. We detect webpack 5 using the same logic as [our dev server](https://github.com/cypress-io/cypress/blob/c0fc23a052e53354a8300dd3f783cb161ae161e1/npm/webpack-dev-server/src/plugin.ts#L116) - we do `if ('NormalModule' in webpack)`, which does not exist in webpack <4.